packages/validator-ajv8/test/harness/compileTestSchema.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * In order to keep things in sync, it may be necessary to run this after making changes in the schemaParser world in
+ * `@rjsf/utils` OR if an AJV update is installed. To run this, simply do the following, starting in the root directory
+ * of the `@rjsf/validator-ajv8` directory:
+ *
+ * - cd test/harness
+ * - node compileTestSchema.js
+ *
+ * Then add the two updated `superSchema.js` and `superSchemaOptions.js` files to your PR
```

Comment:
Hmm...we'd have no idea if something breaks unless we're disciplined about following these steps. Is it burdensome to just run this file and recompile the validators before we run the tests?

Reply:
And if so, we can .gitignore the compiled files, of course

Reply:
Hmmm, good idea, let me see how that goes